src/pytrajplot/plotting/plot_map.py

```diff
@@ -127,10 +167,12 @@ def crop_map(
     domain_boundaries = domain_dict[domain]["domain"]
     ax.set_extent(domain_boundaries, crs=ccrs.PlateCarree(central_longitude=0))
     lon_limits = ax.get_xlim()
-    if np.isclose(np.abs(lon_limits[0] - lon_limits[1]), 360, atol=1):
-        return []
-    else:
-        return domain_boundaries
+
```

Comment:
It is hard to comprehend this message. Under which conditions? Is it really needed?

Reply:
No, it's no longer needed, that's why I commented it out. Also I am not sure whether the condition that was coded here is really what was wanted at the time. Yet I am not sure whether at one point in the future such a feature will be requested again, so for now, I merely commented it out instead of deleting it.

Reply:
I would delete it. You can always use something like: